opentracing/src/main/java/opentracing/TraceCodec.java

```diff
@@ -24,16 +24,16 @@
  * binary message format, so that it can be inspected efficiently by the middleware / routing layers
  * without parsing the whole message.
  */
-public interface TraceContextCodec {
+public interface TraceCodec {
```

Comment:
> By including a codec facility in the API, those ad hoc HTTP calls propagate the TraceContext (or SpanContext, here) from process to process without relying on knowledge of the particular OpenTracing implementation.

So what you're saying is the SPI must define what's required for serialisation of SpanContexts across different implementations to the one tracing platform?

Or are we expecting SpanContexts to go over the wire between different tracing platforms? Because that doesn't make sense to me.